pages/understanding-json-schema/basics.md

```diff
@@ -139,8 +139,8 @@ you control, for example:
 { "$id": "http://yourdomain.com/schemas/myschema.json" }
```

Comment:
Why pushing this change? please remove from the PR anything that is unrelated to this feature.